presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -327,22 +328,33 @@ private QueryResult executeQueryTest()
             }
             while (retry < testTeardownRetries);
             // if teardown is not successful the query fails
-            queryResult = tearDownResult.getState() == State.SUCCESS ? queryResult : tearDownResult;
+            if (tearDownResult.getState() != State.SUCCESS) {
+                queryResult = tearDownResult;
```

Comment:
Should we move this line to after the if? I found the logic easier to comprehend that way.